app/experimenter/kinto/tasks/nimbus.py

```diff
@@ -0,0 +1,164 @@
+def nimbus_push_experiment_to_kinto(experiment_id):
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("push_experiment_to_kinto.started")
+
+    experiment = NimbusExperiment.objects.get(id=experiment_id)
+    if not NimbusBucketRange.objects.filter(experiment=experiment).exists():
+        NimbusIsolationGroup.request_isolation_group_buckets(
+            experiment.slug,
+            experiment,
+            NimbusExperiment.BUCKET_COUNT,
+        )
+
+    data = NimbusExperimentSerializer(experiment).data
+
+    logger.info(f"Pushing {experiment} to Kinto")
+
+    try:
+        kinto_client.push_to_kinto(data)
+
+        experiment.status = NimbusExperiment.Status.ACCEPTED
+        experiment.save()
+
+        generate_nimbus_changelog(experiment, get_kinto_user())
+
+        logger.info(f"{experiment} pushed to Kinto")
+        metrics.incr("push_experiment_to_kinto.completed")
+    except Exception as e:
+        metrics.incr("push_experiment_to_kinto.failed")
+        logger.info(f"Pushing {experiment} to Kinto failed: {e}")
+        raise e
+
+
+@app.task
+@metrics.timer_decorator("check_kinto_push_queue")
+def nimbus_check_kinto_push_queue():
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("check_kinto_push_queue.started")
+
+    queued_experiments = NimbusExperiment.objects.filter(
+        status=NimbusExperiment.Status.REVIEW
+    )
+
+    rejected_collection_data = kinto_client.get_rejected_collection_data()
+    if rejected_collection_data:
+        rejected_slug = kinto_client.get_rejected_record()
+        experiment = NimbusExperiment.objects.get(slug=rejected_slug)
+        experiment.status = NimbusExperiment.Status.DRAFT
+        experiment.save()
+
+        generate_nimbus_changelog(
+            experiment,
+            get_kinto_user(),
+            message=f'Rejected: {rejected_collection_data["last_reviewer_comment"]}',
+        )
+
+        kinto_client.delete_rejected_record(rejected_slug)
+
+    if queued_experiments.exists():
+        if kinto_client.has_pending_review():
+            metrics.incr("check_kinto_push_queue.pending_review")
+            return
+
+        nimbus_push_experiment_to_kinto.delay(queued_experiments.first().id)
+
+        metrics.incr("check_kinto_push_queue.queued_experiment_selected")
+    else:
+        metrics.incr("check_kinto_push_queue.no_experiments_queued")
+
+    metrics.incr("check_kinto_push_queue.completed")
+
+
+@app.task
+@metrics.timer_decorator("check_experiment_is_live")
+def nimbus_check_experiment_is_live():
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("check_experiment_is_live.started")
+
+    accepted_experiments = NimbusExperiment.objects.filter(
+        status=NimbusExperiment.Status.ACCEPTED
+    )
+
+    records = kinto_client.get_main_records()
+    record_ids = [r.get("id") for r in records]
+
+    for experiment in accepted_experiments:
```

Comment:
The task above publishes individually, this one says its to `check_experiment` yet its checking all accepted experiments. I would expect the task to be something like `nimbus_check_experiments_are_live`.